admin/class-automation.php

```diff
@@ -43,6 +43,46 @@ class Automation {
 			'location' => 'article_metadata',
 			'type'     => 'boolean',
 		],
+		'Section'     => [
+			'location' => 'apple_news_api',
```

Comment:
Set this to `article_metadata`
```suggestion
			'location' => 'article_metadata',
```